### ISSUE TITLE
would be nice to be able to increase or decrease font size

### ISSUE THREAD

**Issue by thefonso** [mdh34/quickDocs#78]
**Describe the solution you'd like**
On the DevDocs website I can control+ or control- the entire page to change the font size. 

**Describe alternatives you've considered**
Something like this OR just a setting in preference that would allow the font to be increased and decreased would be nice.

**Additional context**

**Comment by ldelossa** [comment]
Yeah, I downloaded this because Zeal doesn't allow increasing the font of the UI. 
Looks like this doesn't either. 

So tired of squinting at my docs haha.
